app/services/usps_in_person_proofing/transliterator.rb

```diff
@@ -0,0 +1,46 @@
+module UspsInPersonProofing
+  # Container to hold the results of transliteration
+  TransliterationResult = Struct.new(
```

Comment:
to match the Rails autoloader expectations, either:
- this struct should be in its own file `usps_in_person_proofing/transliteration_result.rb`
- it should be nested under the Transliterator as `Usps::InPersonProofing::TransliterationResult`

Reply:
Moved to `UspsInPersonProofing::Transliterator::TransliterationResult`.